pilot/pkg/bootstrap/server.go

```diff
@@ -311,11 +313,20 @@ func NewServer(args *PilotArgs) (*Server, error) {
 		&authenticate.ClientCertAuthenticator{},
 		kubeauth.NewKubeJWTAuthenticator(s.kubeClient, s.clusterID, s.multicluster.GetRemoteKubeClient, spiffe.GetTrustDomain(), features.JwtPolicy.Get()),
 	}
-
-	caOpts.Authenticators = authenticators
 	if features.XDSAuth {
+		if args.JwtRule != "" {
```

Comment:
Move this up. If JwtRule is not set, we still want OIDC auth in the chain for CA.

Reply:
This has been moved up.